liveviz/convertjson.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import csv, json
+from sys import argv
+
+try:
+    filename = argv[1]
+except IndexError:
+    print("No input given")
+    exit()
+
+try:
+    f = open(filename, 'r')
+    blockdata = csv.reader(f, delimiter=',')
+
+    timestamp = []
+    height = []
+    nonce = []
+    diff = []
+
+    for row in blockdata:
+        #print (row)
+        timestamp.append(row[3])
+        height.append(row[1])
+        nonce.append(row[2])
+        diff.append(row[0])
+
+    print ("{")
```

Comment:
Oh, I'm not sure about this. Why aren't you manage all JSON_encoding with json_dumps function?

Reply:
it's just a simple hack to get it in the format echarts needs. pythons json_dump didn't produce the result I needed or maybe I was using it wrong. If you can optimize this to get rid of this mess in a pythonic way while still keeping the same output format I'd be very happy to learn :)